lib/collection/src/lib.rs

```diff
@@ -15,6 +15,7 @@ use crate::operations::types::{CountRequest, CountResult, PointRequest};
 use crate::operations::OperationToShard;
 use crate::shard::remote_shard::RemoteShard;
 use crate::shard::shard_config::{ShardConfig, ShardType};
+use crate::shard::shard_transfer::ShardTransfer;
 use crate::shard::ShardOperation;
```

Comment:
Do you want by any chance to optimize imports? :smile:

Reply:
It seems we decided to use the current import style so I am resolving this comment